.github/workflows/python-publish-client.yml

```diff
@@ -1,9 +1,9 @@
-name: Build and Publish Python Package
+name: Build and Publish Python Client Package
 
 on:
   push:
     tags:
-      - 'XXX*'
+      - '*'
```

Comment:
I think this would trigger the workflow when any tag is pushed. I don't think we want this.

For now, I would just remove the whole trigger from the workflow. We can come back to this when we incorporate the bindings into the release process.